src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -376,7 +379,7 @@ file that was distributed with this source code.
                         {% for paramKey, paramValue in admin.persistentParameters %}
                             <input type="hidden" name="{{ paramKey }}" value="{{ paramValue }}">
```

Comment:
Should not we also try to replace this input?

Reply:
Same here: https://github.com/sonata-project/SonataAdminBundle/pull/7727/files#r822988289